python/graphstorm/graphpartitioning/array_readwriter/parquet.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+import numpy as np
+import pandas as pd
+import pyarrow
+import pyarrow.parquet
+
+from .registry import register_array_parser
+
+
+@register_array_parser("parquet")
+class ParquetArrayParser(object):
+    """Wrapper class for parquet read/write operations."""
+    def __init__(self):
+        pass
+
+    def read(self, path):
+        """ Read operation on a file.
+
+        Argument
+        --------
+        path: str
+            Path of a file from which to read.
+        """
+        logging.debug("Reading from %s using parquet format" % path)
+        metadata = pyarrow.parquet.read_metadata(path)
+        metadata = metadata.schema.to_arrow_schema().metadata
```

Comment:
why do we need `metadata` while csv doesn't need it?

Reply:
Here the metadata has a different meaning compared to csv files. The author expects that the dimensions/shape information can be extracted by reading the parquet file header.